content/docs/learning-guide/tutorials/assets/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+linkTitle: Assets
+title: Process Assets with Scene Settings
+description: Learn to process meshes, textures, characters, animation, and PhysX assets for Open 3D Engine (O3DE).
+---
+
+The meshes, textures, and animations you create for your projects, also known as source assets, must be processed by the **Asset Pipeline** before you can use them in your **Open 3D Engine** projects. The tutorials in this section cover all the basics of the tools you will use to customize how runtime optimized product assets are generated with **Scene Settings**.
+
+| Topic | Description |
+| - | - |
+| [Mesh assets](mesh-assets) | Use Scene Settings to process mesh assets for O3DE. |
```

Comment:
Update name of topic to either "Mesh Processing" (`linkTitle`) or "Customize Mesh Asset Processing" (`title`). 
```suggestion
| [Mesh Processing](mesh-assets) | Use Scene Settings to process mesh assets for O3DE. |
```